clang/lib/CodeGen/CodeGenModule.h

```diff
@@ -1384,8 +1384,8 @@ class CodeGenModule : public CodeGenTypeCache {
   void AddGlobalAnnotations(const ValueDecl *D, llvm::GlobalValue *GV);
 
   /// Emit additional args of the annotation.
-  llvm::Constant *
-  EmitSYCLAnnotationArgs(const SYCLAddIRAnnotationsMemberAttr *Attr);
+  llvm::Constant *EmitSYCLAnnotationArgs(
+      SmallVectorImpl<std::pair<std::string, std::string>> &Pair);
```

Comment:
There is whole vector of pairs.
```suggestion
      SmallVectorImpl<std::pair<std::string, std::string>> &Pairs);
```